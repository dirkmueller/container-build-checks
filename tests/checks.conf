[General]
Vendor=org.opensuse
Registry=registry.opensuse.org
Registry+=registry.example.com

[Tags]
Allowed+=opensuse/*
Blocked+=naughty*
