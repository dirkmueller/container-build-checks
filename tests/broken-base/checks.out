Looking at broken-base.containerinfo
Warning: Using manually defined repositories (obsrepositories:/, obs://devel:evil/) in the image. Only obsrepositories:/ is allowed.
Release: 14.19
Tag: opensuse/tumbleweed:latest
Tag: opensuse/tumbleweed:20210519
Warning: None of the tags are unique to a specific build of the image.
Make sure that at least one tag contains the release.
Warning: Could not find prefixed copy of the org.opensuse.reference label
Warning: Could not determine image specific label prefix, some checks will be skipped.
Error: org.openbuildservice.disturl not set correctly, bug in OBS?
Warning: Label org.opencontainers.image.vendor is not set by the image or any of its bases
Warning: The org.opensuse.reference label (notregistry.opensuse.org/opensuse/nottumbleweed:latest) does not use an allowed registry: registry.example.com,registry.opensuse.org
Warning: The org.opensuse.reference label (notregistry.opensuse.org/opensuse/nottumbleweed:latest) does not refer to an existing tag (opensuse/tumbleweed:latest, opensuse/tumbleweed:20210519)

container-build-checks done. Hints: 0 Warnings: 7 Errors: 1
Warnings found, but those are only fatal in certain projects.
Fatal errors found.
Exited with 1
